openedx/core/djangoapps/programs/tasks.py

```diff
@@ -250,7 +252,16 @@ def post_course_certificate_configuration(client, cert_config, certificate_avail
     response.raise_for_status()
 
 
-@shared_task(bind=True, ignore_result=True)
+# pylint: disable=W0613
```

Comment:
Nit: Since "w0613" is not really human understandable, is there any way we can change this to the human readable version? I think this would be... (`disable=unused-argument`)?